records/2022-04-14.secrets-usage.md

```diff
@@ -123,7 +123,7 @@ table below.
 | Storing notification in Redis / SQS queues                                 | Yes        | No             | The complete notification is signed and stored into the queue.                                                                  | 1. [Signing notification content prior to storage](https://github.com/cds-snc/notification-api/blob/02d431774ced0e2bc1dbe19ee8ceddcd1a8371dc/app/celery/tasks.py#L159-171)                                                                                                                                                                                                                                                                                                                                                                                                                  |   |
-| Password reset code / Verification URL / Confirmation URL / Invitation URL | Yes        | Yes            | Expires after 1 hour.                                                                                                           | 1. [url_with_tokens](https://github.com/cds-snc/notification-api/blob/bdf1174440272f3bc6d05587cb9100edbe12e3a9/app/utils.py#L29-L34)                                                                                                                                                                                                                                                                                                                                                                                                                                                        |   |
+| Password reset code / Verification URL / Confirmation URL / Invitation URL | Yes        | No            | Expires after 1 hour.                                                                                                           | 1. [url_with_tokens](https://github.com/cds-snc/notification-api/blob/bdf1174440272f3bc6d05587cb9100edbe12e3a9/app/utils.py#L29-L34)                                                                                                                                                                                                                                                                                                                                                                                                                                                        |   |
```

Comment:
Just noticed the "Expires after 1 hour" - this is true for everything except the invitation, which expires in 2 days.

Reply:
Changed it!